src/components/staking/Undelegations.vue

```diff
@@ -35,18 +33,11 @@ export default {
     undelegations: {
       query() {
         /* istanbul ignore next */
-        return gql`
-          query pendingundelegations($networkId: String!, $delegatorAddress: String!) {
-            pendingundelegations(networkId: $networkId, delegatorAddress: $delegatorAddress) {
-              ${UndelegationFragment}
-            }
-          }
-        `
+        return UndelegationsForDelegator(this.network)
```

Comment:
we only use this query here. what do you think about putting the query in here directly like we do at other places?

Reply:
Done!